frontends/web/src/routes/account/account.tsx

```diff
@@ -232,9 +231,10 @@ export function Account({
                 {!isAccountEmpty && <ActionButtons {...actionButtonsProps} />}
               </div>
             </div>
-            {showBuyButton && (
+            {isAccountEmpty && (
               <BuyReceiveCTA
                 code={code}
+                exchangeBuySupported={exchangeBuySupported}
                 unit={balance.available.unit}
                 balanceList={[[code, balance]]}
               />
```

Comment:
I think it could also happen in accountsummary AddBuyReceiveOnEmptyBalances in testnet with eth only accounts.

Reply:
Yes accountSummary has the Buy button. I imagine we also wanna hide it here too right whenever it's not applicable / (not supported)? (only makes sense I guess 😄 )
<img width="1488" alt="Screenshot 2023-07-04 at 19 47 48" src="https://github.com/digitalbitbox/bitbox-wallet-app/assets/28676406/0d581681-ccaf-4484-9450-604233717266">

Reply:
the conditions in AddBuyReceiveOnEmptyBalances are a bit hard to read and I am not even 100% sure if it is correct, especially the ` code={balanceList[0][0]}`. 

```js
if (balanceList.map(entry => entry[1].available.unit).every(isBitcoinCoin)) {
    return <BuyReceiveCTA code={balanceList[0][0]} unit={'BTC'} balanceList={balanceList} />;
  }
```

Could you double check if that all makes sense?